README.md

```diff
@@ -7,6 +7,8 @@ A boilerplate to setup and practice Modern JavaScript
 [![Practice Modern JavaScript Download](https://img.shields.io/badge/Download-Practice_Modern_JavaScript_🚀-9055A2.svg?style=flat-square&maxAge=2592000)](https://github.com/abhisekp/Practice-Modern-JavaScript/archive/master.zip)  
 [![Practice Modern JavaScript Tweet](https://img.shields.io/badge/Tweet-Practice_Modern_JavaScript-1da1f2.svg?style=flat-square&maxAge=2592000)](https://twitter.com/intent/retweet?tweet_id=726917916843601920&related=abhisek%3ACreator,babeljs%3ATranspiler)
 
+[![Dependency Status](https://david-dm.org/abhisekp/Practice-Modern-JavaScript.svg)](https://david-dm.org/abhisekp/Practice-Modern-JavaScript) [![devDependency Status](https://david-dm.org/abhisekp/Practice-Modern-JavaScript/dev-status.svg)](https://david-dm.org/abhisekp/Practice-Modern-JavaScript#info=devDependencies)
```

Comment:
**Move it between line 6 and 7** and **Add two spaces at the end of line** so that it appears after *License* but before *Download*.

- Adding two spaces after a line adds a line break (`<br>`) :white_check_mark: 
- Adding a blank line adds a new paragraph (`<p>`) :x:  
  The paragraph comes with a top margin which makes it appear there is a gap towards to the top.

The badges look perfect :white_check_mark: 
Thank you.